src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -238,6 +238,9 @@ protected static function getBundleDefaultConfig()
                 'log' => true,
                 'throw' => true,
             ),
+            'links' => array(
+                'enabled' => !class_exists(FullStack::class),
```

Comment:
What about adding the component as a dev dependency to the `composer.json` file instead?